index.bs

```diff
@@ -4738,6 +4745,41 @@ RogueList when performing ECDAA-Verify (see section 3.6 in [[!FIDOEcdaaAlgorithm
 [[FIDOMetadataService]] provides one way to access such information.
 
 
+## Security Benefits for [RPS] ## {#sctn-rp-benefits}
+
+The main benefits offered to [=[RPS]=] by this specification include:
+
+1. Users and accounts can be secured using widely compatible, easy-to-use multi-factor authentication.
+1. The [=[RP]=] does not need to provision [=authenticator=] hardware to its users. Instead, each user can independently obtain
```

Comment:
Please add benefit of using attestation to understand the authenticator security characteristic.

Reply:
Please elaborate what you mean by that.

Reply:
Fixed.

Reply:
ok.